src/integrations/meilisearch.ts

```diff
@@ -28,5 +43,13 @@ export const searchMeili = async (params: string): Promise<Meili[]> => {
       ...requestOptions,
     },
   );
-  return res.hits;
+  return {
+    pagination: {
+      current: res.hits.length + res.offset + res.limit,
```

Comment:
is not current offset + limit?

Reply:
From my logic current hits are <= limit (depending on if there is a next page), so if you add limit it becomes `next`?

Reply:
Then probably current = offset + hits.length right?

What's already offset + what actually got returned in current query.

Reply:
yeah, that sounds right to me